src/test/java/org/codeforamerica/shiba/pages/UserJourneyPageTest.java

```diff
@@ -289,10 +289,13 @@ void longFileNamesShouldGetTruncated() {
         getToDocumentUploadScreen();
 
         String filename = "very_long_file_name_that_should_truncate.jpeg";
-        String truncatedName = "very_long_file...jpeg";
+        String filenameName = "very_long_file_name_that_should_truncate";
```

Comment:
I think this test could be removed altogether? Doesnt seem like theres a good way to check for the truncated name